static-instrumenter/maven-plugin/src/main/java/io/opentelemetry/contrib/staticinstrumenter/plugin/maven/PackagingSupportFactory.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.staticinstrumenter.plugin.maven;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Enumeration;
+import java.util.Map;
+import java.util.function.Supplier;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+
+public final class PackagingSupportFactory {
+
+  private static final Map<String, Supplier<PackagingSupport>> SUPPORTED_FRAMEWORKS =
+      Map.of(
+          "BOOT-INF/lib/", () -> new PackagingSupport("BOOT-INF/classes/"),
```

Comment:
Just wondering: why are we checking for the existence of `BOOT-INF/lib`, if we want to move files from `BOOT-INF/classes`?

Reply:
Historical reasons I believe. Will change that.